emscripten.py

```diff
@@ -775,7 +776,8 @@ def make_export_wrappers(exports, delay_assignment):
   wrappers = []
   for name in exports:
     mangled = asmjs_mangle(name)
-    if shared.Settings.ASSERTIONS:
+    # emscripten stack functions are called very early on by writeStackCookie
+    if shared.Settings.ASSERTIONS and not name.startswith('emscripten_stack_'):
```

Comment:
maybe add to the comment `and we do not need to instrument them` ? (is that the reason, and if so, why? don't we use these wrappers for various things?)

Reply:
Done